docker/test/integration/.ropeproject/config.py

```diff
@@ -0,0 +1,112 @@
+# The default ``config.py``
```

Comment:
Does this file make sense to include?  Not familiar with rope.  If it has purpose, it does need a license header.